### ISSUE TITLE
server system alias names do not transfer to custom dashboards ?

### ISSUE THREAD

**Issue by centminmod** [nginxinc/nginx-amplify-agent#57]
using latest 0.44-1 agent on CentOS 7.3 64bit

When you change a server system hostname to use an alias name, the alias name isn't reflected in custom dashboards created for that hostname. Instead of showing alias name, the full hostname is shown on custom dashboards created.

**Comment by ptreyes** [comment]
Do you mean the following:

  - custom graphs created with the wizard ("new set")?
  - automatically generated titles with the "long hostname"
  - long hostname changed to alias
  - graph titles not changed

Or was it something else?

**Comment by centminmod** [comment]
When you add a new server, the default dashboard lists the server hostname (long name i.e. host.domain.com). Then create a custom dashboard from new set and it lists `Requests/s - nginx 1.13.1 @ host.domain.com`. Then change server host.domain.com to alias i.e. OVH MC-32 but custom dashboards still shows `Requests/s - nginx 1.13.1 @ host.domain.com`.  Was expecting it to be `Requests/s - nginx 1.13.1 @ OVH MC-32`

**Comment by ptreyes** [comment]
Yeah, that's an interesting case :) It's more like a feature now. We'll see what we could do..

**Comment by ptreyes** [comment]
I've created an internal ticket for that, but that will requires some work, so please stay tuned.
